Source/Resources/Content/MSGraph/Groups/BUILTIN - Corporate Devices.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "This group includes all Corporate Devices, regardless of whether they are physical, virtual, or virtual desktops.",
+  "description": "What is it: A dynamic device group that includes all corporate devices, regardless of whether they are physical, virtual, or virtual desktops. \n Simeon Notes: This group is used to assign apps and policies that should apply to all devices.",
```

Comment:
**What does this do?**
Creates a group that includes all corporate devices regardless if they are virtual, physical or virtual desktops.  

**Why should you use this?**
This group is used to assign applications and policies that should apply to all devices.

**What is the end user impact?**
Users of this group will be assigned all applications and policies. 

**Learn more**
NA